xdsl/dialects/tensor.py

```diff
@@ -683,6 +683,33 @@ class FromElementsOp(IRDLOperation):
     result = result_def(TensorType.constr(ELEMENT_TYPE))
     assembly_format = "$elements attr-dict `:` type($result)"
 
+    traits = traits_def(NoMemoryEffect())
+
+    def __init__(
+        self,
+        elements: Sequence[SSAValue] | SSAValue,
```

Comment:
Could make it variadic? Also let's provide a way to explicitly specify the result type if we already know it
```suggestion
        *elements: SSAValue,
        result_type: Attribute | None = None
```

Reply:
Actually, I think I was too quick to accept. With this change it is no longer possible to specify whether you want to have a ``tensor<i64>`` or a ``tensor<1xi64>``, which is an important distinction for my usecase.

Reply:
The result type would specify i64, or? I'm talking about the shape. It is unclear if giving a single argument gives shape (1,) or shape (,) and even if, then it would not be clear how to specify the other